test/scenario_3/raider_example_3.yaml

```diff
@@ -11,7 +11,7 @@ aoi_group:
 height_group:
   height_levels: 0 100 500 1000
 los_group:
-  ray_trace: True  # Use projected slant delay by default
+  ray_trace: True  
```

Comment:
Thought we would rename this to be either zenith, slant from raytracing or slant from projection options?

Reply:
Yes we should, but I was minimizing changes in this PR. It actually is handled as such in the Python library but the config file and parsing needs to be updated.

Reply:
We refer in the logs to zenith, slant through zenith projection, slant through raytracing. We should make it transparant and easy for users so would say we make that change now as we say we support these use-cases.

Reply:
I suggest replacing ray_trace in the config file as: 
`los_group:
     los_type: Zenith # Chooose from zenith, projected, ray_trace
`

Reply:
We need to be consistent through-out. The suggestion is not consistent with the current pyton LOS group which has: Zenith, Conventional, Raytracing.

This again is not very clear zenith is a direction, slant is a direction.
projection or ray-tracing is a method.